src/main/java/com/twilio/survey/util/AppSetup.java

```diff
@@ -2,79 +2,89 @@
+		if (userInfo.isEmpty()) {
+			String[] credentials = userInfo.split(":");
+			queryStringParams.add(new BasicNameValuePair("username", credentials[0]));
 
-        try {
-            dbUri = new URI(url);
-        } catch (URISyntaxException e) {
-            System.out.println("Unable to parse DB URL");
-        }
-        String[] userInfo = dbUri.getUserInfo().split(":");
-        String username = dbUri.getUserInfo().split(":")[0];
-        String password = null;
-        if (userInfo.length > 1) {
-            password = dbUri.getUserInfo().split(":")[1];
-        } else {
-            password = "";
-        }
+			String password = credentials[1];
+			if (password.isEmpty()) {
+				password = "";
+			}
 
-        String dBUrl =
-                "jdbc:postgresql://" + dbUri.getHost() + ":" + dbUri.getPort() + dbUri.getPath() +
-                        "?user=" + username + "&password=" + password;
-        if (getSslEnabledDb()) {
-            dBUrl += "&ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory";
+			queryStringParams.add(new BasicNameValuePair("password", password));
+		}
 
-        }
-        params.put("url", dBUrl);
+		if (getSslEnabledDb()) {
+			queryStringParams.add(new BasicNameValuePair("ssl", "true"));
+			queryStringParams.add(new BasicNameValuePair("sslfactory", "org.postgresql.ssl.NonValidatingFactory"));
+		}
 
-        return params;
-    }
+		String queryString = URLEncodedUtils.format(queryStringParams, StandardCharsets.UTF_8);
+		String dBUrl = String.format("jdbc:postgresql://%s:%s%s?%s", dbUri.getHost(), dbUri.getPort(), dbUri.getPath(),
+				queryString);
 
-    public int getPortNumber() {
-        String port = env.get("PORT");
+		params.put("url", dBUrl);
 
-        if (port != null) {
-            return Integer.parseInt(port);
-        } else {
-            return 8080;
-        }
-    }
+		return params;
+	}
 
-    public String getDatabaseURL() {
-        return env.get("DATABASE_URL");
-    }
+	public int getPortNumber() {
+		String port = env.get("PORT");
 
-    /**
-     * Fetches the environment variable TWILIO_DISABLE_DB_SSL. This needs to be set to true
-     * if the database doesn't have SSL enabled
-     */
-    public boolean getSslEnabledDb() {
-        String ssl = env.get("TWILIO_DISABLE_DB_SSL");
-        if (ssl != null && ssl.compareToIgnoreCase("true") == 0) {
-            return false;
-        } else {
-            return true;
-        }
-    }
+		if (port != null) {
+			return Integer.parseInt(port);
+		} else {
+			return 8080;
+		}
+	}
+
+	public String getDatabaseURL() {
+		return env.get("DATABASE_URL");
+	}
+
+	/**
+	 * Fetches the environment variable TWILIO_DISABLE_DB_SSL. This needs to be
+	 * set to true if the database doesn't have SSL enabled
+	 */
+	public boolean getSslEnabledDb() {
+		String ssl = env.get("TWILIO_DISABLE_DB_SSL");
+		if (ssl != null && ssl.compareToIgnoreCase("true") == 0) {
```

Comment:
This whole method could be replaced with `return !Boolean.parseBoolean(env.get("TWILIO_DISABLE_DB_SSL"));`